docs/userguide/scheduling/propagate-dependencies.md

```diff
@@ -121,3 +129,324 @@ $  kubectl get configmap
 NAME               DATA   AGE
 my-nginx-config    1      27m
 ```
+
+## Handling Policy Conflicts
+
+When multiple resources share the same dependent resource (for example, two Deployments both use the same ConfigMap),
+and those resources are distributed by different PropagationPolicy objects with conflicting settings, Karmada needs
+to decide how to handle the shared dependency.
+
+### What is a Policy Conflict
+
+A PropagationPolicy conflict happens when:
+
+- Multiple independent resources (e.g., `Deployment A` and `Deployment B`) reference the same dependent resource (e.g., a `ConfigMap`)
+- Each independent resource is matched by a different `PropagationPolicy`
+- Those PropagationPolicy objects set different values for `conflictResolution` or `preserveResourcesOnDeletion`
+
+When this happens, the attached ResourceBinding created for the shared ConfigMap will receive conflicting instructions
+from the referencing ResourceBindings.
+
+### Conflicting Fields
+
+Two fields can conflict:
+
+- **conflictResolution**: Controls how Karmada handles resource conflicts in member clusters
+  - `Overwrite`: Karmada will overwrite the resource if it already exists in the member cluster
+  - `Abort`: Karmada will stop and report an error if the resource already exists
+- **preserveResourcesOnDeletion**: Controls whether resources should be preserved when deleted from Karmada
+  - `true`: Keep the resources in member clusters even after deletion from Karmada
+  - `false`: Remove the resources from member clusters when deleted from Karmada
+
+### How Karmada Resolves Conflicts
+
+Karmada automatically resolves conflicts using simple rules:
+
+- **ConflictResolution**: If any referencing ResourceBinding uses `Overwrite`, the resolved value is `Overwrite`. Otherwise it is `Abort`.
+- **PreserveResourcesOnDeletion**: If any referencing ResourceBinding sets this to `true`, the resolved value is `true`. Otherwise it is `false`.
```

Comment:
Can this be summarized as the rule we previously mentioned: first state the rules, then explain the two strategies separately according to your current writing style?